packages/btleplug/native/src/ble.rs

```diff
@@ -182,18 +272,27 @@ pub fn scan(sink: StreamSink<Vec<BleDevice>>, filter: Vec<String>) -> Result<()>
     send(Command::Scan { sink, filter })
 }
 
+pub fn events(sink: StreamSink<BleEvent>) -> Result<()> {
+    send(Command::Event { sink: sink })
+}
+
 async fn inner_scan(sink: StreamSink<Vec<BleDevice>>, _filter: Vec<String>) -> Result<()> {
```

Comment:
Since filter is used now, the _ should be removed.

Reply:
Done!